src/main/java/uk/gov/companieshouse/accountsDates/AccountsDates.java

```diff
@@ -29,34 +19,34 @@
      * @return
      * 
      */
-    public LocalDate convertStringToDate(String stringDate) {
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
-        return LocalDate.parse(stringDate, formatter);
-    }
-
+    public LocalDate convertStringToDate(String stringDate);
+    
+    /**
+     * 
+     * Takes a {@link String} in isodate time/date format and converts it to a Java 8 {@link LocalDate}
+     * 
+     * @param dateTimeString
+     * @return
+     * 
+     */
+    public LocalDate getLocalDatefromDateTimeString(String dateTimeString);
+    
+    
     /**
      * Takes a Java 8 {@link LocalDate} and converts it to a {@link String}
      * 
      * @param date
      * @return
      */
-    public String convertDateToString(LocalDate date) {
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
-        return date.format(formatter);
-    }
+    public String convertDateToString(LocalDate date);
 
     /**
      * Formats a {@link String} date from format '2017-03-05' to '3 March 2005'
```

Comment:
The formats are implementation specific. Can you add the example (from format '2017-03-05' to '3 March 2005') to the implementation instead please?

Reply:
Resolved in 054c90f